frontend/src/components/DatabaseBackup/utils.ts

```diff
@@ -0,0 +1,60 @@
+import { BackupPlanPolicySchedule, BackupSetting } from "@/types";
+
+export type BackupSettingEdit = Pick<
+  BackupSetting,
+  "enabled" | "dayOfWeek" | "hour" | "retentionPeriodTs"
+>;
+
+export const PLAN_SCHEDULES: BackupPlanPolicySchedule[] = [
+  "UNSET",
+  "WEEKLY",
+  "DAILY",
+];
+
+export const AVAILABLE_DAYS_OF_WEEK = [0, 1, 2, 3, 4, 5, 6];
```

Comment:
nit: does this work?
```
  [...Array(7).keys()];
```

Also, should AVAILABLE_HOURS_OF_DAY be [0, to 23]?

Reply:
@dragonly defined it to be 00:00 to 06:00 am previously. wdyt?

Reply:
I have no preference on this user limitation. [0, ..., 23] LGTM too.

Reply:
updated